languages/python/oso/oso/oso.py

```diff
@@ -90,3 +91,6 @@ def _print_polar_log_message(self):
                 "Polar tracing enabled. Get help with "
                 + "traces from our engineering team: https://help.osohq.com/trace"
             )
+
+    def validate_config(self):
```

Comment:
WDYT about calling this `validate_roles_config()` (or something similar) that indicates that it's only validating the 'Polar roles' predicates instead of all of the user's Polar code?

Reply:
sure